[
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/241936116",
    "html_url": "https://github.com/electron/electron/issues/6951#issuecomment-241936116",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/6951",
    "id": 241936116,
    "user": {
      "login": "kevinsawicki",
      "id": 671378,
      "avatar_url": "https://avatars0.githubusercontent.com/u/671378?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/kevinsawicki",
      "html_url": "https://github.com/kevinsawicki",
      "followers_url": "https://api.github.com/users/kevinsawicki/followers",
      "following_url": "https://api.github.com/users/kevinsawicki/following{/other_user}",
      "gists_url": "https://api.github.com/users/kevinsawicki/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/kevinsawicki/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/kevinsawicki/subscriptions",
      "organizations_url": "https://api.github.com/users/kevinsawicki/orgs",
      "repos_url": "https://api.github.com/users/kevinsawicki/repos",
      "events_url": "https://api.github.com/users/kevinsawicki/events{/privacy}",
      "received_events_url": "https://api.github.com/users/kevinsawicki/received_events",
      "type": "User",
      "site_admin": true
    },
    "created_at": "2016-08-24T01:55:36Z",
    "updated_at": "2016-08-24T01:55:36Z",
    "body": "> executing call via browser returns:\n\nIs the update url behind any authentication or anything? Can you load it in an incognito browser window?\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/241939510",
    "html_url": "https://github.com/electron/electron/issues/6951#issuecomment-241939510",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/6951",
    "id": 241939510,
    "user": {
      "login": "jschimmoeller",
      "id": 1423536,
      "avatar_url": "https://avatars2.githubusercontent.com/u/1423536?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/jschimmoeller",
      "html_url": "https://github.com/jschimmoeller",
      "followers_url": "https://api.github.com/users/jschimmoeller/followers",
      "following_url": "https://api.github.com/users/jschimmoeller/following{/other_user}",
      "gists_url": "https://api.github.com/users/jschimmoeller/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/jschimmoeller/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/jschimmoeller/subscriptions",
      "organizations_url": "https://api.github.com/users/jschimmoeller/orgs",
      "repos_url": "https://api.github.com/users/jschimmoeller/repos",
      "events_url": "https://api.github.com/users/jschimmoeller/events{/privacy}",
      "received_events_url": "https://api.github.com/users/jschimmoeller/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-08-24T02:21:28Z",
    "updated_at": "2016-08-24T02:21:28Z",
    "body": "yes an incognito browser window does return for the update.  not behind authenicate. \n\nI did notice that the url is bad when it comes back from update:\n{\"url\":\"/download/2.5.1/osx_64?filetype=zip\",\"name\":\"2.5.1\",\"notes\":\"\",\"pub_date\":\"2016-08-24T01:08:25.000Z\"}\n\nhere is what it looks on a generic fresh localhost server:\n{\n  \"url\": \"http://localhost/download/2.5.1/osx_64?filetype=zip\",\n  \"name\": \"2.5.1\",\n  \"notes\": \"\",\n  \"pub_date\": \"2016-08-24T02:06:27.000Z\"\n}\n\nbased on this testing; I believe we should just close it; local config issue. \n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/263457173",
    "html_url": "https://github.com/electron/electron/issues/6951#issuecomment-263457173",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/6951",
    "id": 263457173,
    "user": {
      "login": "pconerly",
      "id": 469808,
      "avatar_url": "https://avatars1.githubusercontent.com/u/469808?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/pconerly",
      "html_url": "https://github.com/pconerly",
      "followers_url": "https://api.github.com/users/pconerly/followers",
      "following_url": "https://api.github.com/users/pconerly/following{/other_user}",
      "gists_url": "https://api.github.com/users/pconerly/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/pconerly/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/pconerly/subscriptions",
      "organizations_url": "https://api.github.com/users/pconerly/orgs",
      "repos_url": "https://api.github.com/users/pconerly/repos",
      "events_url": "https://api.github.com/users/pconerly/events{/privacy}",
      "received_events_url": "https://api.github.com/users/pconerly/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-11-29T02:19:08Z",
    "updated_at": "2016-11-29T02:19:08Z",
    "body": "I just hit this too, and I also have `electron-release-server` running behind an ngrok.io proxy.\r\n\r\nSo in `electron-release-server/config/local.js` I had:\r\n\r\n```\r\nmodule.exports = {\r\n  appUrl: 'SITE_URL',\r\n...\r\n```\r\n\r\nWhich is invalid, and ERS smartly defaulted to a relative url.  When I set the `appUrl` to my ngrok endpoint it worked.  Thanks :)"
  }
]